src/cookbook/design/orientation.md

```diff
@@ -135,7 +135,40 @@ class OrientationList extends StatelessWidget {
   <img src="/assets/images/docs/cookbook/orientation.gif" alt="Orientation Demo" class="site-mobile-screenshot" />
 </noscript>
 
+## Locking device orientation
+
+In the previous section, you learned 
+how to adapt the app UI to device orientation changes.
+
+Flutter also allows you define the specific preferred orientations
+that your app supports. You can either:
+
+- lock the app to single orientation like only portrait up position or
+- allow multiple orientations like both portrait up and down, but not landscape.
```

Comment:
I was confused by "portrait up/down", but then got to the link at the end of the section and saw that they're values for `DeviceOrientation`. That should be made clear where we first mention them, so maybe link to `DeviceOrientation` upfront:

```suggestion
Flutter also allows you to specify the orientations your app supports 
using the values of [`DeviceOrientation`]. You can either:

- Lock the app to a single orientation, like only the `portraitUp` position, or...
- Allow multiple orientations, like both `portraitUp` and `portraitDown`, but not landscape.
```